config/users.php

```diff
@@ -47,6 +47,9 @@
             // determines if registration workflow includes email validation
             'validate' => true,
         ],
+        'Login' => [
+            'flashMessage' => false, // bool
```

Comment:
@rochamarcelo Do you think we should leave it as false by default? or do we activate it at once?

Reply:
false is good here